payment_paysera/utils.py

```diff
@@ -6,7 +6,7 @@
 
 
 def make_full_url_getter(env):
-    base_url = env['ir.config_parameter'].sudo().get_param('web.base.url')
+    base_url = env.get_base_url()
```

Comment:
How come you are calling this method directly on `env`?.. It should be on relevant model, like `'payment.acquirer'`. Did you really test this part?

Also it does not really make sense that this method is called getter, because it just returns URL, not some closure function for URLs or something. But lets leave it as is for compatibility purposes:)

Reply:
It was called env but technically it was passing self :)

Reply:
You mean you changed to pass `self`. Cause I see in change. Lets not change passed object and keep original one. Even though its a bit redundant that we have self and then get same thing from `env`.

Reply:
But you need to pass payment acquirer record otherwise it wont work with this new get_base_url

Reply:
I mean it work but it will just returns web.base.url which would defeat the purpose of this new feature